src/helpers/time.ts

```diff
@@ -14,9 +18,94 @@ export function formatCentiseconds(centiTime: number): string {
   return `${s}.${prefix(cs)}`;
 }
 
+/**
+ * Pads out a number with a 0 if it's under 10
+ * @param n
+ */
 function prefix(n: number): string {
   if (n < 10) {
     return `0${n}`;
   }
   return `${n}`;
 }
+
+export const SECOND_IN_CS = 100;
+export const MINUTE_IN_CS = 60 * SECOND_IN_CS;
+export const HOUR_IN_CS = 60 * MINUTE_IN_CS;
+
+interface PluralizeParams {
+  count: number;
+  word: string;
+  options?: {
+    fixed?: number;
+    abbreviate?: boolean;
+  };
+}
+
+/**
+ * Adds an s to a word if count is over 1.
+ * Takes options to pad the count and abbreviate the word with its
+ * first letter
+ * @param count
+ * @param word
+ * @param options
+ */
+export function pluralize({ count, word, options = {} }: PluralizeParams) {
+  const countStr =
+    options.fixed && count % 1 > 0 ? count.toFixed(options.fixed) : count;
+  const countDesc = options.abbreviate
+    ? word[0]
+    : ` ${count === 1 ? word : `${word}s`}`;
+  return countStr + countDesc;
+}
+
+interface CentiSecondsToHumanReadableParams {
+  c: number;
+  options?: {
+    short?: boolean;
+  };
+}
+
+/**
+ * Converts Centiseconds to a human-readable string like "5:30 minutes"
```

Comment:
"5:30 minutes" doesn't actually make much sense...what's the use case here?

I would expect "5 seconds" or "10 minutes" or "5:30" but in English, I would read the original as "5 minutes and 30 seconds minutes"

Localization should somehow be taken into account for a library like this but I'm not sure if localization makes sense here...

Reply:
yeah sorry that comment doesn't make sense, it's 5 minutes and 30 seconds